dtschema/schemas/dt-core.yaml

```diff
@@ -74,7 +74,7 @@ patternProperties:
       - $ref: types.yaml#/definitions/phandle
 
   # property and node namespace overlaps. Catch both here
-  "^[a-zA-Z0-9][a-zA-Z0-9,+\\-._]{0,63}$":
+  "^[a-zA-Z0-9][a-zA-Z0-9#,+\\-._]{0,63}$":
```

Comment:
I don't think '#' is allowed in node names. Or if it is, I don't think there are any cases of it.

Reply:
As the code comment says, the regex applies both to *property* as well as *node* names.

In this case, the change is required for *property* names, not node names. As the commit message explains, I'm having to deal with `ibm,#dma-address-cells` and `ibm,#dma-size-cells` properties which the regex currently doesn't allow. Those properties are defined in <a href="https://git.kernel.org/pub/scm/linux/kernel/git/robh/linux.git/tree/Documentation/devicetree/bindings/security/tpm/ibmvtpm.txt?h=dt/next">Documentation/devicetree/bindings/security/tpm/ibmvtpm.txt</a>. I am converting that file to DT schema <a href="https://lore.kernel.org/all/20230927115300.GA1587935-robh@kernel.org/">per your request</a>, see <a href="https://github.com/l1k/linux/commits/tpm-dt3">here</a>.

Reply:
Ah yes, you are right.